_data/extensions.yaml

```diff
@@ -410,6 +430,13 @@ categories:
         - security
         groupId: io.quarkus
         artifactId: quarkus-elytron-security
+      - name: Elytron Security OAuth 2.0
+        description: "Secure your services using OAuth 2.0"
```

Comment:
This one would be better with `Secure your applications with OAuth2 opaque tokens`

It should be used in very specific cases.